template/iconComponent.ts

```diff
@@ -0,0 +1,50 @@
+interface TemplateProps {
+    fileName: string;
+    data: string;
+}
+
+function componentTemplate ({fileName,data}:TemplateProps) {
+    const render = `
+// GENERATED BY ./scripts/iconScript.ts
+
+import * as React from 'react';
+import { IconProps } from '../helper';
+
+function Eos${fileName}({size = "m", color = "black", rotate = 0}: IconProps) {
+    switch (size) {
```

Comment:
Maybe another way to do this would be to have an object and so something like:

```js
const valuesMap= {
  xs: 4,
  s: 8,
  base: 14,
  m: 18,
  l: 24,
  xl: 32,
  xxl: 48,
  xxxl: 64,
}

size = valuesMap['xs']
// size: 4
```
It might be a little cleaner?

Reply:
For assigning the value you wold ofc use the `size` prop `valuesMap[size]`

Reply:
Yes, this will be a much cleaner approach. I will implement this

Reply:
I have made the required changes, also added a conditional to check if or not the `size` prop falls under any of the given pre defined aliases